apps/site/assets/ts/schedule/components/schedule-finder/TableRow.tsx

```diff
@@ -61,6 +67,11 @@ const BusTableRow = ({
         onClick={onClick}
         onKeyPress={e => handleReactEnterKeyPress(e, onClick)}
       >
+        {anySchoolTrips && (
+          <td className="schedule-table__td--tiny">
+            <strong>{isSchoolTrip && "S"}</strong>
```

Comment:
Can we move the isSchoolTrip outside of the strong tag?

Reply:
sorry -- I didn't get what you were saying in the other comment -- updated!